invenio_search_ui/bundles.py

```diff
@@ -39,6 +47,63 @@
     output='gen/search.%(version)s.css'
 )
 
+def catalog(domain):
+    """TODO."""
+    return os.path.join(
+        resource_filename('invenio_search_ui', 'translations'),
+        '*',
+        '*',
+        '{0}.po'.format(domain),
+    )
+
+
+class I18nBundle(Bundle):
+    """Magically evaluate lazy strings as file names."""
+
+    def _get_contents(self):
+        """Create strings from lazy strings."""
+        def files():
+            for value in super(I18nBundle, self)._get_contents():
+                for path in glob.glob(value):
+                    yield path
+        return list(files())
+
+    contents = property(_get_contents, Bundle._set_contents)
+
+
+class AngularGettextFilter(Filter):
+    name = 'angular-gettext'
+
+    def output(self, _in, out, **kwargs):
+        out.write("angular.module('tindThemeTranslations', ['gettext']).run(['gettextCatalog', function (gettextCatalog) {")
```

Comment:
@jirikuncar is there are way to get the module name?  We can then use this to replace 'tindTheme'.  It should then be easier to do the import of the translations in the angular app based on the module name + 'Translations'